flake.nix

```diff
@@ -187,8 +186,8 @@
                                     ${optionalString useSudo "--use-remote-sudo"}
                                 '' else ''
                                 {{.REMOTE_COMMAND}} {{.REMOTE_SSH_OPTS}} {{.REMOTE_USER}}@{{.REMOTE_HOST}} \
-                                "${optionalString useSudo "{{.REMOTE_SUDO_COMMAND}} {{.REMOTE_SUDO_OPTS}}"} nixos-rebuild {{.REBUILD_ACTION}} \
-                                --flake '{{.REMOTE_CONFIG_DIR}}#{{.HOSTNAME}}'"
+                                  "${optionalString useSudo "{{.REMOTE_SUDO_COMMAND}} {{.REMOTE_SUDO_OPTS}}"} nixos-rebuild {{.REBUILD_ACTION}} \
+                                  --flake '{{.LOCAL_FLAKE_SOURCE}}#{{.HOSTNAME}}'"
```

Comment:
I understand that this will work, because the hash is the same on both the remote and the local store. Neverthanless it seems a bit hacky to do it like this, or at least the naming of the variable is confusing/wrong. I would prefer to have it point to to a symlink created by the deploy action somewhere in `config-dir`